runtime/moonbeam/src/governance/councils.rs

```diff
@@ -58,4 +61,7 @@ impl pallet_collective::Config<OpenTechCommitteeInstance> for Runtime {
 	type WeightInfo = moonbeam_weights::pallet_collective_open_tech_committee::WeightInfo<Runtime>;
 	type SetMembersOrigin = referenda::GeneralAdminOrRoot;
 	type MaxProposalWeight = MaxProposalWeight;
+	type KillOrigin = EnsureRoot<AccountId>;
+	type DisapproveOrigin = EnsureRoot<AccountId>;
```

Comment:
@stiiifff We need to ask to the foundation what they want these origins to be. Maybe `referenda::GeneralAdminOrRoot`?

Reply:
Change to `FastGeneralAdminOrRoot`.